packages/renderer/src/lib/statusbar/TaskIndicator.svelte

```diff
@@ -12,14 +14,31 @@ let title: string | undefined = $derived.by(() => {
   return `${runningTasks.length} tasks running`;
 });
 
+// single task (if only one task is running)
+let singleCurrentTask = $derived.by(() => {
+  if (runningTasks.length !== 1) return undefined;
```

Comment:
nit: would be god to have a test `multiple cancellable tasks should not display cancel`.

Reply:
I agree

Reply:
test added